build/run_tests.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-xvfb-run ctest --output-on-failure
+xvfb-run ctest -j8 --output-on-failure
```

Comment:
sorry, I shouldn't have included this in the commit.  However, it would be nice if at least the travis CI used -j2 since the travis runs on two cores, I believe.